src/stream/pipeline/runner.rs

```diff
@@ -115,16 +115,14 @@ impl PipelineRunner {
                                     && current_previous_position == position
                                 {
                                     lost_timestamps += 1;
-                                    warn!("Position did not change {lost_timestamps}");
```

Comment:
Is there a reason to remove this one ? IIRC that's kind of necessary to identify some cameras that can have problems

Reply:
Yeah, good one, I'm trying to remove repetitive logs that don't actually add value. It already has a log when it normalizes, I'll add a log when it starts too, then we'll have all the necessary information.